### ISSUE TITLE
Recording selectors

### ISSUE THREAD

**Issue by mkorpela** [MarketSquare/robotframework-browser#801]
**Is your feature request related to a problem? Please describe.**
Selectors and selector strategies are one of more complex problems in Browser UI automation.
There is a big jump from manually clicking an element and figuring out how to detect it from dom.

**Describe the solution you'd like**
I propose that we add a custom selector strategy that can pause and capture a selector.
Selector would be captured first by the test creator clicking the element. This would happen so that the test would pause on the operation with `__DETECT__` selector. Mouse hover would show some select visualisation. Click (with approval) would log the recorded selector and run the keyword.

```RobotFramework
*** Test Cases ***
Recording selector
    Click  __DETECT__
    Get Text  __DETECT__   ==   expected
```

UPDATE:
Went with this:
```RobotFramework
*** Test Cases ***
Recording selector
    ${selector}=  Record selector
    Get Text  ${selector}  ==   expected
```

**Comment by mkorpela** [comment]
https://stackoverflow.com/a/32100212/308189

**Comment by mkorpela** [comment]
https://github.com/fczbkk/css-selector-generator-benchmark
Maybe this for real use: https://github.com/antonmedv/finder

**Comment by mkorpela** [comment]
Simpler maybe to just have a magic selector like DETECT that stops and logs

**Comment by mkorpela** [comment]
https://github.com/MarketSquare/robotframework-browser/pull/882
Working on this here.

Still missing:

- [ ] highlight potential select area
- [ ] accept by clicking highlight element
- [ ] show discovered css selector on the page
- [ ] show test step in question

**Comment by mkorpela** [comment]
Decided to make first minimal solution with a keyword `Record selector`.
That records the selector that is under mouse after 15 seconds.
Also during that time console.logs all the seen selectors.

**Comment by mkorpela** [comment]
Fixed in #882
